codesamples/tests.py

```diff
@@ -16,9 +16,13 @@ def setUp(self):
             is_published=False)
 
     def test_published(self):
-        self.assertQuerysetEqual(CodeSample.objects.published(),
-                                 ['<CodeSample: Copy One>'])
+        expected = ['<CodeSample: Copy One>']
+        published_qs = CodeSample.objects.published()
+        actual = [f'<CodeSample: {str(obj)}>' for obj in published_qs]
+        self.assertEqual(actual, expected)
```

Comment:
Perhaps this would simpler using `assertCountEqual()`, like this:

```suggestion
        self.assertCountEqual(CodeSample.objects.published(),[self.sample2])
```

I find the name of the method slightly misleading, but it would work nicely here, I think:

> Test that sequence first contains the same elements as second, regardless of their order

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertCountEqual

The only difference I see is that we would no longer cover the `__str__` of the model. That could be separate test, if it's deemed important.

Reply:
It's possible to rewrite those tests like this:
```python
    def test_published(self):
        self.assertQuerySetEqual(CodeSample.objects.published(),
                                ['<CodeSample: Copy One>'], transform=repr)
```